src/Data/Fin/Substitution/Example.agda

```diff
@@ -15,11 +15,11 @@ module Data.Fin.Substitution.Example where
 Please see README.Data.Fin.Substitution.UntypedLambda instead."
 #-}
 
-open import Data.Fin.Substitution
-open import Data.Fin.Substitution.Lemmas
+open import Data.Fin.Substitution using (Lift; Sub; Subs; Application; TermSubst)
+open import Data.Fin.Substitution.Lemmas using (TermLemmas)
 open import Data.Nat.Base hiding (_/_)
 open import Data.Fin.Base using (Fin)
-open import Data.Vec.Base
+open import Data.Vec.Base using (Vec; _∷_; []; lookup)
```

Comment:
These are (non-essential) changes to a *deprecated* module.
It's generally better *not* to do this unless you have made other changes which would cause this to break.

That said, please note the module pointed to *instead*: `README.Data.Fin.Substitution.UntypedLambda`

IIRC, modules under `doc/README/` aren't rechecked under CI, unless `doc/README.agda` has been updated to reflect changes, so it's worth investigating propagating your commits to the above module instead of this one.